components/alertbanner/stories/template.js

```diff
@@ -0,0 +1,82 @@
+import { html } from 'lit-html';
+import { classMap } from 'lit-html/directives/class-map.js';
+
+import { Template as Divider } from '@spectrum-css/divider/stories/template.js';
+import { Template as Button } from '@spectrum-css/button/stories/template.js';
+import { Template as CloseButton } from "@spectrum-css/closebutton/stories/template.js";
+import { Template as Icon } from '@spectrum-css/icon/stories/template.js';
+
+
+import '../index.css';
+
+export const Template = ({
+  rootClass = "spectrum-AlertBanner",
+  isOpen = true,
+  text,
+  variant,
+  customClasses = [],
+...globals
+}) => {
+  const { express } = globals;
+
+  try {
+    if (!express) import(/* webpackPrefetch: true */ "../themes/spectrum.css");
+    else import(/* webpackPrefetch: true */ "../themes/express.css");
+  } catch (e) {
+    console.warn(e);
+  }
+  const iconName =
+  variant === "negative"
+    ? "Alert"
+    : variant === "info"
+    ? "Info"
+    : "";
+
+  return html`
+  <div class=${classMap({
+    [rootClass]: true,
+    'is-open': isOpen,
+    [`${rootClass}--${variant}`]: typeof variant !== "undefined",
+    ...customClasses.reduce((a, c) => ({ ...a, [c]: true }), {}),
+  })}>
+    <div class="spectrum-AlertBanner-body">
+      <div class="spectrum-AlertBanner-content">
+        ${iconName ? Icon({
+          ...globals,
+          iconName,
+          customClasses: [
+            `${rootClass}-icon`,
+          ],
+        }) : ""}
+        <p class="spectrum-AlertBanner-text">${text}</p>
+      </div>
+      ${Button({
+        size: 'm',
+        variant: "staticWhite",
+        treatment: "outline",
+        label: 'Action',
+        customClasses: [`${rootClass}-action-button`]
+      })}
+    </div>
+    <div class="spectrum-AlertBanner-end">
+    ${Divider({
+      vertical: true,
+      size: 's',
+      customClasses: [`${rootClass}-divider`, `${rootClass}--vertical`, `${rootClass}-divider--staticWhite`],
+      tag: 'div',
+      style: {
```

Comment:
You have this set as `verticalStyle` in the Divider template so we should probably pick one! I think style is probably best to just have it available whether the divider is vertical or not

Reply:
my thought was that these styles should only be used when vertical and I was trying to make that clear. Do you think it is more confusing this way?

Reply:
ok I think I am following you're suggesting that it should be `style` so it can be more flexible and other components can pass a style to the `divider`. 

In this case if the divider is vertical, as per the docs for the component, it needs the inline styling `align-self: stretch; height: auto`  So I don't really want it flexible. I refactored it to:
`style=${vertical === true ? "min-height: 20px; height: auto; align-self: stretch" : ""}`

This way I am also not passing around something that should always be the same if its vertical.
This might need more refactoring as divider is used more broadly but for now it seems like the simplest approach.